gen3utils/deployment_changes/generate_comment.py

```diff
@@ -53,6 +53,12 @@
     "cedar-wrapper": "cedar-wrapper-service",
     "kayako-wrapper": "kayako-wrapper-service",
     "frontend-framework": "gen3-frontend-framework",
+    "metadata-delete-expired-objects": "sower-jobs",
+    "ssjdispatcher.job_images.indexing": "indexs3client",
+    "_regex": {
+        "^sower\..*\.pelican-.*$": "pelican",
```

Comment:
do we have to worry about this regex being seen as escape character in python? Just looking at the pytest warning

Reply:
oops, i wanted to address those and i forgot. i just pushed a fix